doc/control.rst

```diff
@@ -114,10 +116,10 @@ Control system synthesis
 
     acker
     create_statefbk_iosystem
-    dlqr
     h2syn
     hinfsyn
     lqr
+    dlqr
```

Comment:
I understand why this might go here, but doesn't it make more sense to list alphabetically? 

Alternatively, we could list this as `lqe, dlqe`, since they go together?

Note that if you pass a `LTI` object to `lqr` (and `lqe`), the function will check the timebase to determine whether to use the continuous time or discrete time version of the function.

Reply:
re-alphebetized them in recent commit.